SwiftKitFFM/src/main/java/org/swift/swiftkitffm/SwiftFFM.java

```diff
@@ -12,28 +12,23 @@
 //
 //===----------------------------------------------------------------------===//
 
-package org.swift.swiftkit;
+package org.swift.swiftkitffm;
 
-import org.swift.swiftkit.util.PlatformUtils;
+import org.swift.swiftkitcore.SwiftHeapObject;
+import org.swift.swiftkitcore.util.PlatformUtils;
 
-import java.io.File;
-import java.io.IOException;
 import java.lang.foreign.*;
 import java.lang.invoke.MethodHandle;
 import java.lang.invoke.MethodHandles;
 import java.lang.invoke.VarHandle;
-import java.nio.file.CopyOption;
-import java.nio.file.FileSystems;
-import java.nio.file.Files;
-import java.nio.file.StandardCopyOption;
 import java.util.Arrays;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
-import static org.swift.swiftkit.util.StringUtils.stripPrefix;
-import static org.swift.swiftkit.util.StringUtils.stripSuffix;
+import static org.swift.swiftkitffm.util.StringUtils.stripPrefix;
+import static org.swift.swiftkitffm.util.StringUtils.stripSuffix;
 
-public class SwiftKit {
+public class SwiftFFM {
```

Comment:
I think we can call this type `SwiftRuntime` now as it is the runtime entrypoints we call using ffm... No need to call it FFM in the type -- it is in the ffm module (or rather `.foreign` module)